docs/protocol_overview.md

```diff
@@ -0,0 +1,245 @@
+  The notion of *responsibility* and *power* will be explained [later](#consensus-leader).
+
+## Summary
+
+Here's a summarizd version of the Simperby protocol.
+
+1. We want the node operation to be **simple and lightweight** to realize a truly distributed, decentralized and self-hosted organization.
+2. To be so, it is assumed that most of the nodes are **rarely online** and the protocol produces new **blocks on-demand**.
+3. To realize that, the **consensus round must be very long**.
+4. For every block, members can **vote on or propose an agenda, propagating** to each other.
+5. At the same time, members can also **propagate their chats** used for the communication, which is **coordinated by the block proposer**.
+6. If there is a **governance-approved (majority-voted) agenda** on the network, the block proposer should **include it in the block along with the chat log**, and propose to the consensus.
+7. In that the block proposer should be online most of the time (chat coordination and block proposing) this **responsibility should be laid on few of the validators most ot the time**, to ensure the rarely-online assumption.
+8. Also the role of **block proposer has some authorities** (chat coordination and agenda inclusion) that can't be cryptographically verified if misused (typically censorship).
+9. Thus there exists **'few of the validators' that can be lazy or commit harmful misuses** of their authorities which aren't really byzantine faults or invalid blocks.
+10. Normally this isn't a problem, if the rounds are short and the block proposer changes regularly, but we're not.
+11. We need a special mechanism to **'veto' the block proposer not to waste the long rounds, in the consensus layer**. Thus we introduce a special variation of Tendermint called *Vetomint*.
+12. In Vetomint, validators can veto the current block proposer, but still the round will progress (changing the block proposer) if all the honest validators either vote or veto.
+
+## Simple and Light Node Operation
+
+Simperby strongly encourages each member to run their own node,
+in order to ensure that the network is decentralized and distributed (and so truly self-hosted). In other words, running a Simperby chain is not about using AWS servers by a few people, but using their own laptops to physically run the chain. To realize that, it is important to **keep the node operation as simple and lightweight as possible**.
+
+### Rarely-online nodes
+
+One of the most important condition for a 'simple' and 'lightweight' node operations is how often the node needs to be online.
+If we want to make the validators run nodes on their personal laptops, for example, it is not realistic to assume that they will be online 24/7.
+
+Since Simperby's BFT consensus assumes an partially-synchronous network, rarely-online nodes can be trivially handled because it's no more than one typical case of an asynchrony, if
```

Comment:
```suggestion
Since Simperby's BFT consensus assumes a partially-synchronous network, rarely-online nodes can be trivially handled because it's no more than one typical case of an asynchrony, if
```